### PR TITLE
Update examples titles for SEO

### PR DESCRIPTION
**Pull request by yirenlu92**
- Update titles of some examples for SEO

### CODE REVIEW

**Comment by charlesfrye** [comment]
🚀 The docs preview is ready! Check it out here:     https://modal-labs-examples--frontend-preview-b642b59.modal.run

**Comment by sandcat100** [comment]
is it intentional that the titles don't match w what's in the sidebar?

File: 05_scheduling/hackernews_alerts.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # lambda-test: false
 # tags: ["featured"]
 # ---
-# # Hacker News Slackbot
+# # Cron job in the Cloud to search Hacker News
```
Comment (charlesfrye) [inline 05_scheduling/hackernews_alerts.py:5]:
```suggestion
# # Run cron jobs in the cloud to search Hacker News
```

File: 06_gpu_and_ml/llm-structured/jsonformer_generate.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # ---
 # lambda-test: false
 # ---
-# # Generate synthetic data with Jsonformer
+# # Structured output generation with JSONFormer
```
Comment (charlesfrye) [inline 06_gpu_and_ml/llm-structured/jsonformer_generate.py:4]:
```suggestion
# # Structured output generation with Jsonformer
```

File: 06_gpu_and_ml/stable_diffusion/flux.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 # tags: ["use-case-image-video-3d", "featured"]
 # ---
 
-# # Run Flux fast with `torch.compile` on Hopper GPUs
+# # Run Flux fast on H100s
```
Comment (charlesfrye) [inline 06_gpu_and_ml/stable_diffusion/flux.py:7]:
```suggestion
# # Run Flux fast on H100s with `torch.compile`
```

File: 06_gpu_and_ml/text-to-video/mochi.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 # cmd: ["modal", "run", "--detach", "06_gpu_and_ml/text-to-video/mochi.py", "--num-inference-steps", "64"]
 # ---
 
-# # Generate videos from text prompts with Mochi
+# # Text-to-video AI with Mochi
```
Comment (charlesfrye) [inline 06_gpu_and_ml/text-to-video/mochi.py:5]:
```suggestion
# # Text-to-video generation with Mochi
```

File: 06_gpu_and_ml/yolo/finetune_yolo.py [hunk 5]
```diff
@@ -2,7 +2,7 @@
 # args: ["--no-quick-check"]
 # tags: ["use-case-image-video-3d", "use-case-finetuning"]
 # ---
-# # Fine-Tuning and Inference for Computer Vision with YOLO
+# # Object detection with YOLO open source model
```
Comment (charlesfrye) [inline 06_gpu_and_ml/yolo/finetune_yolo.py:5]:
```suggestion
# # Fine-tune open source YOLO models for object detection
```

File: 06_gpu_and_ml/llm-serving/vllm_inference.py [hunk 6]
```diff
@@ -5,7 +5,7 @@
 # tags: ["use-case-lm-inference", "featured"]
 # ---
 
-# # Run an OpenAI-Compatible vLLM Server
+# # Fast LLM inference with Llama 3.1-8B and vLLM
```
Comment (charlesfrye) [inline 06_gpu_and_ml/llm-serving/vllm_inference.py:8]:
```suggestion
# # Ru OpenAI-compatible LLM inference with LLaMA 3.1-8B and vLLM
```

**Review by charlesfrye — CHANGES_REQUESTED** [review]

**Comment by charlesfrye** [comment]
> is it intentional that the titles don't match w what's in the sidebar?

Yes. We use different titles in the headers, sidebar, and featured examples/use-case pages. They have slightly different purposes (e.g. the sidebar is scanned by humans but the headers are indexed by search engines, including our docs search).

**Review by charlesfrye — APPROVED** [review]

**Comment by charlesfrye** [comment]
🚀 The docs preview is ready! Check it out here:     https://modal-labs-examples--frontend-preview-29dfb3e.modal.run

**Comment by charlesfrye** [comment]
🚀 The docs preview is ready! Check it out here:     https://modal-labs-examples--frontend-preview-c273b89.modal.run

**Pull request merged**